qemu/tests/balloon_hotplug.py

```diff
@@ -37,6 +39,19 @@ def run_pm_test(pm_test, plug_type):
                               % (pm_test, plug_type), logging.info)
         utils_test.run_virt_sub_test(test, params, env, pm_test)
 
+    def enable_balloon_service(session):
+        """
+        Install balloon service and check its status in windows guests
+        :param session: shell Object
+        """
```

Comment:
can we add this step in doc string of run(test, params, env)?

Reply:
Sure, added.
Thanks